src/payloads.atd

```diff
@@ -1,82 +1,83 @@
 type json <ocaml module="Yojson.Basic"> = abstract
 
-type capped_string = string wrap <ocaml module="Capped_string">
+type capped_string_4k = string wrap <ocaml module="Capped_string_4k">
+type capped_string_512 = string wrap <ocaml module="Capped_string_512">
 type datetime = string wrap <ocaml module="Datetime">
 type platform = string wrap <ocaml module="Platform">
 type severity_level = string wrap <ocaml module="Severity_level">
 type uuid = string wrap <ocaml module="Uuid">
 
 type sdk_info =
-  { name : capped_string
-  ; version: capped_string
-  ; ?integrations : capped_string list nullable }
+  { name : capped_string_512
+  ; version: capped_string_512
+  ; ?integrations : capped_string_512 list nullable }
 
 type mechanism =
-  { type_ <json name="type"> : capped_string
-  ; ?description : capped_string nullable
-  ; ?help_link : capped_string nullable
+  { type_ <json name="type"> : capped_string_512
+  ; ?description : capped_string_512 nullable
```

Comment:
shouldn't this one be `capped_string_4k`?

Reply:
Right, fixed that and also the cap on the exception "value".